project/findbugs-exclude-filters.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<!--
+  Copyright 2016 LinkedIn Corp.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+
+<!-- FindBugs exclusion filters -->
+<FindBugsFilter>
+  <!-- Exclude scala files -->
+  <Match>
+      <Source name="~.*\.scala" />
```

Comment:
Shouldn't it exclude all files except .java .  Why to specifically mention scala (what if we have python files )